delivery.yaml

```diff
@@ -39,3 +47,19 @@ pipeline:
   - type: docs
     name: skipper
     path: site
+- id: ARM builds
+  overlay: ci/golang
+  type: script
+  commands:
+  - desc: build-push
+    cmd: |
+      if [[ $CDP_TARGET_BRANCH == master && ! $CDP_PULL_REQUEST_NUMBER ]]; then
```

Comment:
There is native CDP support for this logic with events.

Reply:
I think we have to do it anyways manual, because the first merge to master will not create a nicely tagged release version. We have to retrigger the pipeline a second time. I think I will add an additional check to it, to only burn CPU cycles if it makes sense.